doc/api/addons.markdown

```diff
@@ -44,7 +44,7 @@ be used as a starting-point for your own Addon.
 This "Hello world" example is a simple Addon, written in C++, that is the
 equivalent of the following JavaScript code:
 
-    module.exports.hello = function() { return 'world'; };
+    module.exports.hello = () => 'world';
```

Comment:
Huh, I thought we unintended all code blocks in the docs recently, but i looks like we missed some.

Want to give it another pass, @eljefedelrodeodeljefe?